Makefile

```diff
@@ -411,3 +411,14 @@ container-build-and-push: container-build container-push ## Build and push all t
 .PHONY: cluster-functest 
 cluster-functest: ginkgo ## Run e2e tests in a real cluster
 	./hack/functest.sh $(GINKGO_VERSION)
+
+.PHONY: super-linter
+super-linter: ## Runs super linter locally (Supported Linters -> https://github.com/github/super-linter#supported-linters)
+	docker run --rm \
+		-e RUN_LOCAL=true \
+		-e USE_FIND_ALGORITHM=true \
+		-e IGNORE_GITIGNORED_FILES=true \
+		-e FILTER_REGEX_EXCLUDE="/vendor/" \
+		-v $$(pwd):/tmp/lint \
+		-w /tmp/lint \
```

Comment:
Needs to be set to overcome the current bug - https://github.com/github/super-linter/issues/3786